torch_geometric/explain/explainer.py

```diff
@@ -64,8 +66,15 @@ def get_prediction(self, *args, **kwargs) -> torch.Tensor:
 
         with torch.no_grad():
             out = self.model(*args, **kwargs)
-        if self.model_config.mode == ModelMode.classification:
+        if self.model_config.mode == ModelMode.multiclass_classification:
             out = out.argmax(dim=-1)
+        elif self.model_config.mode == ModelMode.binary_classification:
+            if self.model_config.return_type.value == 'raw':
+                out = (out > 0).long()
+            if self.model_config.return_type.value == 'probs':
+                out = (out > 0.5).long()
+            elif self.model_config.return_type.value == 'log_probs':
+                out = (out > math.log(0.5)).long()
```

Comment:
I don't think the combination of `binary_cross_entropy` and `log_probs` is valid - can we disable this combination during `ModelConfig` initialization?